platformio.ini

```diff
@@ -140,8 +139,6 @@ lib_ignore =
     HALSim
     HALTarget
     Webots
-extra_scripts =

```

Comment:
Lines 459 and 460 must also be removed

Reply:
I will keep the other line for future use.